components/Layout.tsx

```diff
@@ -56,6 +59,68 @@ function Logo({ title }: { title: string }) {
   );
 }
 
+function LanguageToggle() {
+  const router = useRouter();
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(
+    null,
+  );
+  const open = Boolean(anchorEl);
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <div>
+      <Button
+        id='language-toggle-button'
+        aria-controls={open ? 'language-toggle-menu' : undefined}
+        aria-haspopup='true'
+        aria-expanded={open ? 'true' : undefined}
+        onClick={handleClick}
+        endIcon={<KeyboardArrowDownIcon />}
+        sx={{
+          background: 'none',
+          color: '#fff',
+        }}
+      >
+        {router.locale}
+      </Button>
+      <Menu
+        id='language-toggle-menu'
+        anchorEl={anchorEl}
+        open={open}
+        onClose={handleClose}
+        onClick={handleClose}
+        disableScrollLock
+        sx={{
+          '& .MuiMenuItem-root': {
+            padding: 0,
+            '& a': {
+              padding: '6px 16px',
+              textAlign: 'center',
+              textDecoration: 'none',
+            },
+          },
+        }}
+      >
+        {(router.locales || []).map((locale) => (
+          <MenuItem key={locale}>
+            <Link
+              href={router.asPath}
+              locale={locale}
+            >
+              {locale}
```

Comment:
Can we capitalize these here in the list as well?

Reply:
updated